From d2dc4312ddc174bb86f57b040c4a6c61bdf2ada0 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@st.com>
Date: Wed, 23 Feb 2022 10:54:23 +0100
Subject: [PATCH 0642/1695] plat-stm32mp1: conf: enable HSI/CSI clock
 calibration

Set CFG_STM32_CLKCALIB to 'y' in conf.mk.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 279390cd2f9e9e88812ed3562523398e355f5f10 plat-stm32mp1: conf: enable HSI/CSI clock calibration]
Signed-off-by: Gatien Chevallier <gatien.chevallier@st.com>
Change-Id: If16c185ebbddab556b652c69fd3df294c76deb8f
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/240970
Reviewed-by: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Lionel DEBIEVE <lionel.debieve@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/286639
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/plat-stm32mp1/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index 9a1ceb4c0..a29f9b419 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -260,6 +260,7 @@ endif
 $(call force,CFG_STM32MP1_SCMI_SHM_SIZE,0x1000)
 
 CFG_STM32_BSEC ?= y
+CFG_STM32_CLKCALIB ?=y
 CFG_STM32_CRYP ?= y
 CFG_STM32_ETZPC ?= y
 CFG_STM32_GPIO ?= y
-- 
2.47.2

